download_test_vectors.sh

```diff
@@ -9,13 +9,16 @@
 
 set -Eeuo pipefail
 
-VERSIONS=(
-  "v1.6.0-alpha.0"
-)
+# Override versions, eg: TEST_VECTORS_VERSIONS=nightly
+if [[ -n "${TEST_VECTORS_VERSIONS:-}" ]]; then
```

Comment:
maybe `CONSENSUS_SPEC_TEST_VECTORS` or (a bit more concisely, I guess `TEST` is implied) `CONSENSUS_TEST_VECTORS`? there are also slashing protection test vectors.

Reply:
Renamed to `CONSENSUS_TEST_VECTOR_VERSIONS`